app/controllers/groups_controller.rb

```diff
@@ -201,12 +201,14 @@ def assign_student_and_next
       if params[:s_id].present?
         student = current_course.students.find(params[:s_id])
       end
+      student_name = params[:names].gsub('(inactive)', '').strip
```

Comment:
1. Let's be a bit more strict here: the exact string is `' (inactive)'` is appended to the name (Line 190), so (1) here we can match `' (inactive)'` exactly. Also you can use a regex to match the end of string `$`, to prevent accidental matches earlier. Finally, since we only want zero or one match, it seems [`String#sub`](https://docs.ruby-lang.org/en/3.4/String.html#class-String-label-Substitution+Methods) is more appropriate.
2. We must have missed this originally, but please replace all instances of the string `'(inactive)'` with an internationalized string. Seems like there's one occurrence on Line 190 and one in our test files.

Reply:
Good point. I have converted this value to an internationalized string.